system_tests/bigquery.py

```diff
@@ -90,7 +92,15 @@ def test_update_dataset(self):
         after = [grant for grant in dataset.access_grants
                  if grant.entity_id != 'projectWriters']
         dataset.access_grants = after
-        dataset.update()
+
+        # We should try and keep `delay` and tries as low as possible to
+        # reduce test running time. tries=3 and delay=30 worked consistenly
+        # when updating a dataset.
```

Comment:
There isn't much content here. Essentially equivalent to "Delay bad. This works."

Any idea why they work? Any idea why we can't go lower?